beets/util/__init__.py

```diff
@@ -735,7 +735,7 @@ def open_anything():
     return base_cmd
 
 
-def interactive_open(target, command=None):
+def interactive_open(targets, command=None):
     """Open the file `target` by `exec`ing a new command. (The new
```

Comment:
The docstring should refer to `targets` (or perhaps the variable should now be named `args` or something else generic?).